pandas/tests/arithmetic/test_datetime64.py

```diff
@@ -599,15 +577,18 @@ def test_comparison_tzawareness_compat(self, op, box_df_fail):
         with pytest.raises(TypeError, match=msg):
             op(dz, np.array(list(dr), dtype=object))
 
-        # Check that there isn't a problem aware-aware and naive-naive do not
-        # raise
+        # Check that there isn't a problem aware-aware and naive-naive do not raise
```

Comment:
I'm not sure what this comment means.  Might be a wording or grammar thing.

Reply:
will re-word, thanks